client/src/components/LoginForm/LoginForm.tsx

```diff
@@ -40,14 +40,15 @@ const LoginForm = (): JSX.Element => {
     //custom styling is implemented using styled components
     <LoginFormStyles>
       <div className="login-container">
-        <label>Username</label>
+        <label className="form-input-label" id="header">Please enter your Login details below:</label>
```

Comment:
This needs to be a p or h1 tag rather than a label then you don't need to use a class name and can use css selectors instead

Reply:
The skinnybanner below already provide this info so this label is unnecessary